staging/src/k8s.io/component-base/cli/flag/ciphersuites_flag_test.go

```diff
@@ -86,18 +86,18 @@ func TestConstantMaps(t *testing.T) {
 		if strings.HasPrefix(declName, "VersionTLS") {
 			discoveredVersions[declName] = true
 		}
-		if strings.HasPrefix(declName, "TLS_RSA_") || strings.HasPrefix(declName, "TLS_ECDHE_") ||
-			strings.HasPrefix(declName, "TLS_AES_") || strings.HasPrefix(declName, "TLS_CHACHA20_") {
```

Comment:
this drops ciphers that were just added in https://github.com/kubernetes/kubernetes/pull/90843, please revert this change